Lib/venv/scripts/nt/activate.bat

```diff
@@ -1,13 +1,11 @@
 @echo off
 
-rem This file is UTF-8 encoded, so we need to update the current code page while executing it
-for /f "tokens=2 delims=:" %%a in ('"%SystemRoot%\System32\chcp.com"') do (
-    set "_OLD_CODEPAGE=%%a"
-)
+rem This file is UTF-8 encoded, so we need to update the current code page while executing it.
+for /f %%a in ('%~dp0python.exe -s -S -E -c "import ctypes; print(ctypes.windll.kernel32.GetConsoleOutputCP());"') do (set "_OLD_CODEPAGE=%%a")
+
 if defined _OLD_CODEPAGE (
-    "%SystemRoot%\System32\chcp.com" 65001 > nul
+    %~dp0python.exe -s -S -E -c "import ctypes; ctypes.windll.kernel32.SetConsoleOutputCP(65001);"
```

Comment:
Just a nit - I don't think the trailing semicolon is required - it's a separator and not a terminator. Same applies below, of course.

Reply:
haha. ctypes pulled me to the dark side ;-). Your comment made my day "just a nit" is a lovely expression. Thanks.